index.bs

```diff
@@ -442,24 +449,23 @@ To <dfn>resolve the requested features</dfn> given |requiredFeatures| and |optio
   1. For each |feature| in |requiredFeatures| perform the following steps:
     1. If |feature| is not a valid [=feature name=], return <code>false</code>.
     1. If the functionality described by |feature| is not supported by |session|'s [=XRSession/XR Device=], return <code>false</code>.
-    1. If the functionality described by |feature| [=requires consent=], append it to |consentRequired|.
+    1. If the functionality described by |feature| requires [=explicit consent=], append it to |consentRequired|.
```

Comment:
This is a bit quirky and we might need to look at other spec text to get it right... That said, I think what we're trying to say here (and also below) is that if implied consent is sufficient and the UA has determined to reject it, then it should be rejected.  Otherwise, if explicit consent is required, then prompt.